sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -825,6 +825,8 @@ class MLPClassifier(ClassifierMixin, BaseMultilayerPerceptron):
         validation score is not improving by at least tol for
         ``n_iter_no_change`` consecutive epochs. The split is stratified,
         except in a multilabel setting.
+        If early stopping is activated, the validation score is used to stop
+        the neural network. Otherwise, the loss is used.
```

Comment:
The above already describes what is happening for the `True` case.

```suggestion
        If early stopping is false, the training loss is used for early
        stopping.
```

Reply:
Here is another suggestion:

```
If early_stopping is False, then the training stops when the training loss does not improve by more than tol for n_iter_no_change consecutive passes over the training set.
```